messaginginapp/src/main/java/io/customer/messaginginapp/ModuleMessagingInApp.kt

```diff
@@ -1,70 +1,30 @@
 package io.customer.messaginginapp
 
 import android.app.Application
-import androidx.annotation.VisibleForTesting
 import io.customer.messaginginapp.di.gistProvider
-import io.customer.sdk.CustomerIOConfig
-import io.customer.sdk.android.CustomerIO
 import io.customer.sdk.communication.Event
 import io.customer.sdk.communication.subscribe
 import io.customer.sdk.core.di.SDKComponent
 import io.customer.sdk.core.module.CustomerIOModule
 import io.customer.sdk.data.request.MetricEvent
-import io.customer.sdk.di.CustomerIOComponent
 
-class ModuleMessagingInApp
-@VisibleForTesting
-internal constructor(
-    override val moduleConfig: MessagingInAppModuleConfig = MessagingInAppModuleConfig.default(),
-    private val overrideDiGraph: CustomerIOComponent?
+class ModuleMessagingInApp(
+    config: MessagingInAppModuleConfig
```

Comment:
I used `config` as customers who were using named parameters will need to change otherwise. Also, I think `config` here is simpler than `moduleConfig` for customers. But open to use `moduleConfig` directly if we think that's not adding much value.